source/shared/net/guides/guide-ledger.rst

```diff
@@ -0,0 +1,148 @@
+
+#. Enter a name for your identity, and then enter a name for your initial account. Select **Continue**.
+
+#. Select an identity provider.
+
+#. Connect your Ledger device to your computer if you haven't done so already and enter your PIN code on the Ledger. Press the buttons above the up and down arrows to choose a digit, and then press both buttons to select the digit.
+
+#. In the Desktop Wallet, there’s a message asking you to open the Concordium application on the Ledger. On the Ledger, press both buttons when it says **Concordium**. The Ledger says **Concordium is ready**.
+
+#. In the Desktop Wallet, there’s a message saying the Ledger is ready, depending. Select **Submit**.
+
+#. Press both buttons to confirm the following on the Ledger:
+
+   - **Create credential**: the number that is assigned to the credential is displayed.
+
+   - **Public key**: verify that the public key on the Ledger corresponds to the public key in the Desktop Wallet. Use the right button to navigate through the key.
+
+#. In the Desktop Wallet, select **Continue**.
+
+#. The Ledger says **Review identity provider info**. Press both buttons, and then press the right button to navigate through the public key and verify that it corresponds to the information in the Desktop Wallet. Press both buttons to confirm.
+
+#. Verify that the **Signature threshold** on the Ledger corresponds to the threshold in the Desktop Wallet.
+
+#. Press the right button, and then press both buttons to sign the identity provider information.
+
+#. In the Desktop Wallet, the **New identity** page is displayed. Enter the identity verification information and select **Submit**. This information will vary depending on the identity provider. The identity provider submits the identity to the blockchain. When the initial account is created in a finalized block on the blockchain, the identity provider confirms the identity, and after confirmation, a green checkmark is displayed next to the Concordium logo on the identity.
+
+#. Select **Finished**. When you’ve created more accounts, you can recognize your initial account by the text *identity* next to the name.
+
+View the account balance
+========================
+
+To view the account balance in the Concordium Desktop Wallet do the following:
+
+- In the Desktop Wallet, go to **Accounts**, and then select the account whose balance you want to see. You can now see the following account information:
+
+  - **Account Total** shows the total number of GTU on the account
+
+  - **Balance** shows the GTU that are at your disposal, the amount of GTU that you’ve staked, and any GTU that are locked in a release schedule.
+
+  - **Shielded balance** shows the amount of GTU that you’ve shielded.
+
+  - **Staked** shows the amount of GTU you've staked if you're node is :ref:`baking <baker-concept>` blocks.
```

Comment:
```suggestion
  - **Staked** shows the amount of GTU you've staked if you're node is :ref:`baking <baker-concept>` blocks.
  
  - **At disposal** is the amount of GTU that is available for use on the balance, hence excluding any GTU locked in a stake or a release schedule.
```
I'm not sure if this suggestion is too technical, since it mentions release schedules, but I felt like it had to be explained what "at disposal" means. What do you think, @LotteKh?

Reply:
Maybe this and **Staked** should also be moved above **Shielded balance** so it matches the order it is shown on the card?

Reply:
I've added links to stake and schedule topics, but I don't think this is the right place to explain the concepts since it's a getting started topic.